src/components/svgs/stations/chongqingrt2021-int.tsx

```diff
@@ -0,0 +1,409 @@
+    nameOffsetY: NameOffsetY;
+    isRapid: boolean;
+    isWide: boolean;
+    wideDirection: 'vertical' | 'horizontal';
+    rapidColor: Theme;
+}
+
+const defaultChongqingRT2021IntStationAttributes: ChongqingRT2021IntStationAttributes = {
+    ...defaultStationAttributes,
+    nameOffsetX: 'right',
+    nameOffsetY: 'top',
+    isRapid: false,
+    isWide: false,
+    wideDirection: 'horizontal',
+    rapidColor: [CityCode.Chongqing, 'cq10', '#5f249f', MonoColour.white],
+};
+
+const ChongqingRT2021IntAttrsComponent = (props: AttrsProps<ChongqingRT2021IntStationAttributes>) => {
+    const { id, attrs, handleAttrsUpdate } = props;
+    const { t } = useTranslation();
+    const fields: RmgFieldsField[] = [
+        {
+            type: 'input',
+            label: t('panel.details.stations.common.nameZh'),
+            value: (attrs ?? defaultChongqingRT2021IntStationAttributes).names[0],
+            onChange: val => {
+                attrs.names[0] = val.toString();
+                handleAttrsUpdate(id, attrs);
+            },
+            minW: 'full',
+        },
+        {
+            type: 'textarea',
+            label: t('panel.details.stations.common.nameEn'),
+            value: (attrs ?? defaultChongqingRT2021IntStationAttributes).names[1],
+            onChange: val => {
+                attrs.names[1] = val.toString();
+                handleAttrsUpdate(id, attrs);
+            },
+            minW: 'full',
+        },
+        {
+            type: 'select',
+            label: t('panel.details.stations.common.nameOffsetX'),
+            value: (attrs ?? defaultChongqingRT2021IntStationAttributes).nameOffsetX,
+            options: { left: 'left', middle: 'middle', right: 'right' },
+            disabledOptions: attrs?.nameOffsetY === 'middle' ? ['middle'] : [],
+            onChange: val => {
+                attrs.nameOffsetX = val as NameOffsetX;
+                handleAttrsUpdate(id, attrs);
+            },
+            minW: 'full',
+        },
+        {
+            type: 'select',
+            label: t('panel.details.stations.common.nameOffsetY'),
+            value: (attrs ?? defaultChongqingRT2021IntStationAttributes).nameOffsetY,
+            options: { top: 'top', middle: 'middle', bottom: 'bottom' },
+            disabledOptions: attrs?.nameOffsetX === 'middle' ? ['middle'] : [],
+            onChange: val => {
+                attrs.nameOffsetY = val as NameOffsetY;
+                handleAttrsUpdate(id, attrs);
+            },
+            minW: 'full',
+        },
+        {
+            type: 'switch',
+            label: t('panel.details.stations.chongqingRT2021Int.isRapid'),
+            oneLine: true,
+            isChecked: (attrs ?? defaultChongqingRT2021IntStationAttributes).isRapid,
+            onChange: val => {
+                attrs.isRapid = val as boolean;
+                handleAttrsUpdate(id, attrs);
+            },
+            minW: 'full',
+        },
+        ...((attrs.isRapid
```

Comment:
There is a `hidden` key RmgFieldsField.

https://github.com/railmapgen/rmp/blob/325f55b45dd5b17f7d9ad5845562293e0ff129ab/src/components/svgs/stations/gzmtr-int-2024.tsx#L370